src/main/resources/assets/changed/lang/hu_hu.json

```diff
@@ -510,9 +516,14 @@
   "eyestyle.changed.v2": "Vékony",
   "eyestyle.changed.tall": "Magas",
 
+  "changed.config.color_picker_red": "Vörös: %s",
+  "changed.config.color_picker_green": "Zöld: %s",
+  "changed.config.color_picker_blue": "Kék: %s",
+
```

Comment:
These changed will be duplicated when I merge the branches.

Reply:
My bad, I assumed your other PR had these changes too, but the grab mechanic won't be merged into master until for 0.14, meaning these changes wont be in 0.13.1